static/scripts/calendar.js

```diff
@@ -55,20 +55,22 @@ $(document).ready(function () {
         editable: false,
         timezone: 'UTC',
         events: function (start, end, timezone, callback) {
+            if('serviceWorker' in navigator){
+                navigator.serviceWorker.addEventListener('message', function (event) {
+                    if (event.origin !== 'https://schul-cloud.org' || 'http://localhost:3100')
```

Comment:
this should come from the cofig, as we run on other domains as well

Reply:
maybe we just can remove that check. this is a litte security check but the check for data.tag should be enough as well

Reply:
i am ok with removing

Reply:
solved: we can auto detect this